mlip_arena/models/externals/pet.py

```
@@ -0,0 +1,31 @@
+from mlip_arena.models.utils import get_freer_device
+
+
+
+with open(Path(__file__).parents[1] / "registry.yaml", encoding="utf-8") as f:
+    REGISTRY = yaml.safe_load(f)
+
+
+class PET_OAM_XL(UPETCalculator):
+    def __init__(
+        self,
+        checkpoint=REGISTRY["PET(OAM)"]["checkpoint"],
```

Ruff (0.14.14)

[warning] Unused method argument: `checkpoint`

(ARG002)